core/src/main/java/org/javarosa/core/model/ImageRestrictionExtension.java

```diff
@@ -0,0 +1,43 @@
+package org.javarosa.core.model;
+
+import org.javarosa.core.util.externalizable.DeserializationException;
+import org.javarosa.core.util.externalizable.ExtUtil;
+import org.javarosa.core.util.externalizable.PrototypeFactory;
+
+import java.io.DataInputStream;
+import java.io.DataOutputStream;
+import java.io.IOException;
+
+/**
+ * Represents an extension to the "upload" question type, for setting a maximum
+ * dimension for the size of an image capture
+ *
+ * @author amstone
+ */
+public class ImageRestrictionExtension implements QuestionDataExtension {
```

Comment:
One question here: Do we want to make this slightly more expressive? IE should this be something like:

ImageCaptureExtension

or something that makes it clear that it's cool for people to continue adding other image question data here?